Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -527,33 +520,26 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
             {
                 // Scenario 1: the listener can clearly understand the message
                 result = perceivedMessage;
-                wrappedMessage = Loc.GetString("chat-manager-entity-whisper-wrap-message",
-                    ("color", language.Color ?? Color.Gray),
-                    ("entityName", name),
-                    ("message", FormattedMessage.EscapeText(result)));
+                wrappedMessage = WrapWhisperMessage(source, "chat-manager-entity-whisper-wrap-message", name, perceivedMessage, language);
```

Comment:
wrappedMessage = WrapWhisperMessage(source, "chat-manager-entity-whisper-wrap-message", name, perceivedMessage, language);

we should send result not perceivedMessage